drivers/gles3/shader_compiler_gles3.cpp

```diff
@@ -862,6 +862,8 @@ ShaderCompilerGLES3::ShaderCompilerGLES3() {
 	actions[VS::SHADER_SPATIAL].renames["CLEARCOAT_GLOSS"] = "clearcoat_gloss";
 	actions[VS::SHADER_SPATIAL].renames["ANISOTROPY"] = "anisotropy";
 	actions[VS::SHADER_SPATIAL].renames["ANISOTROPY_FLOW"] = "anisotropy_flow";
+	actions[VS::SHADER_SPATIAL].renames["DEPTH"] = "gl_FragDepth";
+	//actions[VS::SHADER_SPATIAL].renames["SSS_SPREAD"] = "sss_spread";
```

Comment:
Why add commented out SSS_SPREAD? Is that a rebase oversight?

The order should stay the same between `shader_compiler_gles3.cpp` and `servers/visual/shader_types.cpp`. Maybe put it next to `DEPTH_TEXTURE` in both files?

Reply:
Yeah, SSS spread probably came from rebasing. Will fix the order :)